packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -39,6 +39,8 @@ const tracestateTagKeyFilter = /[^\x21-\x2b\x2d-\x3c\x3e-\x7e]/g
 // Tag values in tracestate replace ',', '~' and ';' with '_'
 const tracestateTagValueFilter = /[^\x20-\x2b\x2d-\x3a\x3c-\x7d]/g
 const invalidSegment = /^0+$/
+const ddParentIdTagKey = '_dd.parent_id'
```

Comment:
Nit: There's a tags.js file that lists common tag names. Should entries like these be listed in that file? /cc @bengl